index.bs

```diff
@@ -3747,25 +3747,25 @@ Record=] of type <code>throw</code>, |exception|, is given by:
 [=Remote end definition=] and [=local end definition=]
 
 <pre class="cddl local-cddl remote-cddl">
-ScriptResult = (
-  ScriptResultSuccess //
-  ScriptResultException
+ScriptCallResult = (
```

Comment:
I wonder if it would be easier to avoid having the typedefs below for each command and just calling this `ScriptEvaluateResult`

Reply:
Done